src/components/picture/index.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import { Children } from "react";
+import Source from "./_children/source";
+import Image from "../image";
+
+const COMPONENT_CLASS_NAME = "c-picture";
+const Picture = ({ children, className, ...rest }) => (
+	<picture
+		{...rest}
+		className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
+	>
+		{/* `Image` and `Source` are the only two supported Children of `Picture` */}
+		{Children.toArray(children).filter((child) => [Image, Source].includes(child.type))}
+	</picture>
+);
+
+Picture.Source = Source;
```

Comment:
For consistency we could map the `Image` to be a child component like 
```
Picture.Image = Source
```

Reply:
Sounds good! I've updated Source and the Storybook documentation to account for the change.